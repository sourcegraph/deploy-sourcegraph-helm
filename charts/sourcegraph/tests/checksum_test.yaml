---
suite: checksumTest
templates:
- frontend/sourcegraph-frontend.Deployment.yaml
tests:
- it: should update checksum when pgsql secret is modified
  set:
    pgsql:
      auth:
        existingSecret: "my-secret"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
- it: should update checksum when minio secret is modified
  set:
    minio:
      auth:
        password: "newpassword"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
- it: should update checksum when codeintel-db secret is modified
  set:
    codeIntelDB:
      auth:
        newkey: "blank"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
- it: should update checksum when codeinsights-db secret is modified
  set:
    codeInsightsDB:
      auth:
        username: "newuser"
        database: "anotherdb"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
- it: should update checksum when redis-store secret is modified
  set:
    redisStore:
      connection:
        endpoint: "external-redis:6379"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
- it: should update checksum when redis-cache secret is modified
  set:
    redisCache:
      connection:
        endpoint: "external-redis:6379"
  asserts:
    - matchSnapshot:
          path: spec.template.metadata.annotations.[checksum/auth]
