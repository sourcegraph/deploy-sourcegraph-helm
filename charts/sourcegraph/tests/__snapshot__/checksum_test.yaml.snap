should update checksum when codeinsights-db secret is modified:
  1: |
    0214987aff8c4e6baba1dfe773e8c3218f80f810e499405ae011960ba86dcbfb
should update checksum when codeintel-db secret is modified:
  1: |
    2518fab3d953a21ea78057f3ad7143f41ec960c0b3b96c49f807ba0fd73ba4b0
should update checksum when minio secret is modified:
  1: |
    bac4d406514d2f8cd604f566607dfc36b963bc324770428d6fd6e8a442037980
should update checksum when pgsql secret is modified:
  1: |
    b960ae6db8c72ca9248278a63593de67f1ddfb9805f82eded969ffde767406db
should update checksum when redis-cache secret is modified:
  1: |
    3253fcbe6996c20f8e3ce7470a0d5843da49f8f55b4402d309504cb0c63568bb
should update checksum when redis-store secret is modified:
  1: |
    3253fcbe6996c20f8e3ce7470a0d5843da49f8f55b4402d309504cb0c63568bb
